clients/java/pom.xml

```diff
@@ -41,7 +35,7 @@
       <dependency>
         <groupId>org.camunda.bpm</groupId>
         <artifactId>camunda-bom</artifactId>
-        <version>${camunda.version}</version>
+        <version>${project.version}</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
```

Comment:
I cannot comment below this line, so I will put it here:
Below, the license plugin and the license header profile are defined. This should not be necessary anymore, as this should be inherited by the parents already.

Reply:
💯